src/period/index.js

```diff
@@ -11,22 +11,25 @@ const { logPeriod } = require('../utils/debug');
 
 module.exports = (bridgeState, sender) => async (
   rsp,
-  chainInfo,
-  chainHeight
+  state,
+  chainInfo
 ) => {
-  const height = chainHeight - 32;
+  const height = chainInfo.height - 32;
+
+  logPeriod('checkBridge');
 
   if (height <= 0 || !bridgeState.previousPeriod) {
     // genesis height doesn't need check
     rsp.status = 1;
     return;
   }
 
-  logPeriod('checkBridge');
+  logPeriod('checkBridge. Period: ', bridgeState.previousPeriod.merkleRoot());
+
   // use a timeout to relax race conditions (if period submission is fast)
   await new Promise(resolve => {
     setTimeout(async () => {
-      await submitPeriodVote(bridgeState.previousPeriod, bridgeState, sender);
```

Comment:
Out of curiosity, I see that with this PR we haven't changed `\src\period\submitPeriodVote.js` - which expect 4 params, instead of 3. Was this code calling `submitPeriodVote` with wrong arguments? 🤔